homeassistant/components/nest/manifest.json

```diff
@@ -11,5 +11,6 @@
   "codeowners": [
       "@awarecan",
       "@allenporter"
-  ]
+  ],
+  "quality_scale": "silver"
```

Comment:
I would say it's platinum.
```suggestion
  "quality_scale": "platinum"
```

- You don't need to disable entities by default if they are all relevant. Some integrations expose uninteresting or highly specialized data that should be opt-in, not opt-out. 
- PARALLEL_UPDATES is to limit if Home Assistant calls all update methods / services at the same time or in sequence. This defaults to unlimited parallel if the entity has `async_update` and 1 request at a time if it's not there. Your entities do not have an async_update because it's not based on polling. So it's 1 at a time. I don't know if Nest API limits parallel requests. Set it to 0 if they do, otherwise leave as-is.
- Aiohttp is not required for custom protocols like Google streaming pub/sub. So this one is ✅

Reply:
The limits are all in terms of queries per minute: 
https://developers.google.com/nest/device-access/project/limits#sandbox_user_limits
so not really about how many requests are in parallel at once so it sounds like this may not be something to add.

More context: During testing, i have seen push back from the API when i change around climate a bunch to get it to test out updates, while i also have cameras refreshing streams.   One thing I considered doing was adding jitter to the internal camera refresh to smooth out background request to smooth streams, but i haven't seen this be a problem during regular use.